### PR TITLE
feat(DatePicker): Add isFutureDateSelectable and initialVisibleMonth props

### PR DESCRIPTION
**Pull request by ayeshakmaz**
Summary:
Add `isFutureDateSelectable` and `initialVisibleMonth` props
Add `isOutsideRange` function to allow for custom checking function

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,13 +5,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+    - Add `isOutsideRange` property to allow for a custom validation function for dates
+    - Add `initialVisibleMonth` property to set the starting month
 
 ## 44.3.1 - 2019-07-30
 - [Patch] Revert **Table** css styles and add **CodeDiff** css styles ([#1206](https://github.com/optimizely/oui/pull/1206))
 
 ## 44.3.0 - 2019-07-25
 - [Patch] Have CSSWhat run on `yarn build-styles` ([#1203](https://github.com/optimizely/oui/pull/1203))
-- [Patch] Update **Table** style to include hover by default in OUI; Update **CodeDiff** src/components/Table/TR/index.js style to be more resiliant inside a table ([#1202](https://github.com/optimizely/oui/pull/1202))
+- [Patch] Update **Table** style to include hover by default in OUI; Update **CodeDiff** src/components/Table/TR/index.js style to be more resilient inside a table ([#1202](https://github.com/optimizely/oui/pull/1202))
```
Comment (daverau-optimizely) [inline CHANGELOG.md]: Nice catch! 👀

File: src/components/DatePicker/overrides.scss [hunk 2]
```diff
@@ -49,6 +49,14 @@
   color: $smoke-dark;
 }
 
+.CalendarDay__today {
+  text-decoration: underline;
```
Comment (daverau-optimizely) [inline src/components/DatePicker/overrides.scss]: Curious if you think extending `.underline` is overkill here.

Comment (ayeshakmaz) [reply src/components/DatePicker/overrides.scss]: How do you mean?

**Comment by benjaminpadula** [comment]
Can we get this merged today?

I suppose since Change History is still unreleased in development and not having this doesn't cause any JS errors, maybe it's fine, but would be a bonus to get it merged :)

**Comment by ayeshakmaz** [comment]
@benjaminpadula Let me resolve the conflicts and merge it in

**Pull request merged**